src/librustdoc/html/render/mod.rs

```diff
@@ -1710,6 +1710,7 @@ fn print_sidebar(cx: &Context<'_>, it: &clean::Item, buffer: &mut Buffer) {
             write!(
                 buffer,
                 "<div class=\"block version\">\
+                     <div class=\"narrow-helper\"></div>
```

Comment:
Missing backslash at the end of the line.

Reply:
thanks